tests/integration/conftest.py

```diff
@@ -38,13 +38,188 @@ def tags():
 
 @pytest.fixture(scope="session")
 def resource_group():
-    yield "rg-idem-inttest-" + "".join(
+    yield "idem-rg-inttest-" + "".join(
         random.choice(string.ascii_lowercase + string.digits) for _ in range(20)
     )
 
 
+@pytest.fixture(scope="session")
+def vm():
+    yield "idem-vm-" + "".join(
+        random.choice(string.ascii_lowercase + string.digits) for _ in range(5)
+    )
+
+
 @pytest.fixture(scope="session")
 def keyvault():
-    yield "kv-idem-inttest-" + "".join(
+    yield "idem-kv-" + "".join(
+        random.choice(string.ascii_lowercase + string.digits) for _ in range(8)
+    )
+
+
+@pytest.fixture(scope="session")
+def test_keyvault():
+    yield "idem-test-kv-" + "".join(
+        random.choice(string.ascii_lowercase + string.digits) for _ in range(8)
+    )
+
+
+@pytest.fixture(scope="session")
+def test_key():
+    yield "idem-test-key-" + "".join(
+        random.choice(string.ascii_lowercase + string.digits) for _ in range(8)
+    )
+
+
+@pytest.fixture(scope="session")
+def key():
+    yield "idem-key-" + "".join(
+        random.choice(string.ascii_lowercase + string.digits) for _ in range(8)
+    )
+
+
+@pytest.fixture(scope="session")
+def storage_account():
+    yield "idemacc" + "".join(
+        random.choice(string.ascii_lowercase + string.digits) for _ in range(16)
+    )
+
+
+@pytest.fixture(scope="session")
+def storage_container():
+    yield "idem-container" + "".join(
+        random.choice(string.ascii_lowercase + string.digits) for _ in range(32)
+    )
+
+
+@pytest.fixture(scope="session")
+def log_analytics_workspace():
+    yield "idem-law-" + "".join(
+        random.choice(string.ascii_lowercase + string.digits) for _ in range(32)
+    )
+
+
+@pytest.fixture(scope="session")
+def postgresql_server():
+    yield "idem-psql-server-" + "".join(
+        random.choice(string.ascii_lowercase + string.digits) for _ in range(8)
+    )
+
+
+@pytest.fixture(scope="session")
+def vnet():
+    yield "idem-vnet-" + "".join(
+        random.choice(string.ascii_lowercase + string.digits) for _ in range(8)
+    )
+
+
+@pytest.fixture(scope="session")
+def vnet2():
+    yield "idem-vnet2-" + "".join(
+        random.choice(string.ascii_lowercase + string.digits) for _ in range(8)
+    )
+
+
+@pytest.fixture(scope="session")
+def subnet():
+    yield "idem-subnet-" + "".join(
+        random.choice(string.ascii_lowercase + string.digits) for _ in range(8)
+    )
+
+
+@pytest.fixture(scope="session")
+def test_vnet():
+    yield "idem-test-vnet-" + "".join(
```

Comment:
I'm counting three vnet fixtures so far. why do we need so many? (haven't looked through all of the code so far)